commerce_option.info.yml

```diff
@@ -0,0 +1,5 @@
+name: Commerce Option
+description: Automates attribute creation.
```

Comment:
I think we need to have a bit more clarity on the description as to exactly what this module does. It actually provides an alternative to attributes right. 

Basically the gist of this module is so that we stop generating variations based on the attributes. It resolves the need for SKU generation based on variations which can get cumbersome if the client has lots of different attributes. Instead, we will only have one parent variation for the product that does not contain any attributes. The attributes will then need to be presented to the user as order item options which will be stored on the Order item entity and displayed as options on the product.

So, I don't know, something like, "This module allows store managers to offer products with multiple options while maintaining a single SKU. It provides an alternative to attribute based variations."

Reply:
I like the description. Never good at coming up with those.